pipeline/src/resources/ColorAnnotator.py

```diff
@@ -0,0 +1,59 @@
+
+import tornado.web
+import socket
+import tornado.ioloop
+import json
+
+
+class Annotator(tornado.web.RequestHandler):
+    def __init__(self):
+        self._addr = ''
+        self._port = 12345
+        self._stream = None
+
+    def _initialize(self, sock):
+        client_socket, _ = sock.accept()
+        self._stream = tornado.iostream.IOStream(client_socket)
+        # self._stream.connect((self._addr, self._port))
+
+    def _handle_request(self, sock, fd, events):
+        self._initialize(sock)
+        self._stream.read_until(b'}', callback=self._annotate)
+
+    def _annotate(self, data):
+        resp = self.annotate(json.loads(data))
```

Comment:
The Python version I have on my VM wants these bytes `decode()`ed - fixed in #20